### ISSUE TITLE
update pgspecial to use parametrized named queries

### ISSUE THREAD

**Issue by msztolcman** [dbcli/pgcli#627]
Few months ago I added to pgspecial new feature: parametrized named queries. It would be nice to have this feature enabled in pgcli.

**Comment by j-bennet** [comment]
Noted. I'll release new version of pgspecial for that.

**Comment by msztolcman** [comment]
@j-bennet Thanks!

**Comment by j-bennet** [comment]
pgspecial 1.7.0 is now released to pypi: https://github.com/dbcli/pgspecial/pull/27. There's been a few PR merges after pgcli 1.3.1, and I think pgcli will be ready for another release after #624 is merged.

**Comment by j-bennet** [comment]
Available in 1.5.0.
